docs/workflow/debugging/libraries/debugging-packages.md

```diff
@@ -96,7 +96,7 @@ ResolvePkgProjReferences:
 To run a test from a single Build Pivot combination, specify all properties and build the `csproj`:
 
 ```
-dotnet msbuild System.Net.ServicePoint.Tests.csproj /t:rebuildandtest /p:TargetGroup=netcoreapp2.0 /p:OuterLoop=true /p:xunitoptions=-showprogress /p:ConfigurationGroup=Debug
+dotnet msbuild System.Net.ServicePoint.Tests.csproj /t:rebuildandtest /p:BuildTargetFramework=netcoreapp2.0 /p:OuterLoop=true /p:xunitoptions=-showprogress /p:Configuration=Debug
```

Comment:
`dotnet build` again here, and no need to pass in the Configuration here, that's just confusing.

Reply:
Also, can we use TargetFramework and not BuildTargetFramework?

Reply:
> Also, can we use TargetFramework and not BuildTargetFramework? 

Do we want people to use TargetFramework here because we will have to specify the targetFramework with the OSGroup as well. We still don't have a check to verify that if a given targetFramework is a supported one.